tests/query/parser/test_expressions.py

```diff
@@ -9,6 +9,113 @@
 from snuba.query.parser.expressions import parse_aggregation
 
 test_data = [
+    (
+        ["f(a *   b, g(c * 3))", None, None],
```

Comment:
one minor nit on your test cases. For unit tests it is advisable to have specific tests that test individual interesting inputs and then, if needed, some more complex tests that put things together. This helps us getting a better sense of the logical coverage (how well the test cover the interesting cases) we have quickly.
In this case I would advise you to have one or two specific test to stress the spaces in the expressions, and just rely on a standard spacing pattern for the others.

Reply:
Good point. Removed some cases of extra spacing.